.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
+  - 1.4
+  - tip
+
+before_install:
+  # install linting tools
+  - go get github.com/golang/lint/golint
+  - go get golang.org/x/tools/cmd/vet
+  - go get github.com/fzipp/gocyclo
+  # install code coverage tools
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
+
+install:
+  - cd lib/goose
+  - go get -t
+  - go build
+  - cd ../..
+
+script:
+  # make sure code is properly formatted and simplified
+  - gofmt -l -s .
+  # check for possible uh ohs
+  - go vet ./...
+  # run tests checking for race conditions
+  - cd cmd/goose
+  - go test ./... -v -race
+  # report test coverage
+  - $HOME/gopath/bin/goveralls -service=travis-ci -repotoken=$COVERALLS
```

Comment:
We should have the above `go test` generate a coverprofile so we're not running the entire test suite twice.